components/org.wso2.carbon.business.rules.web/src/components/auth/Login.jsx

```diff
@@ -188,6 +194,33 @@ export default class Login extends React.Component {
                         >
                             Login
                         </Button>
+                        <br />
+                        <br />
+                        <div style={styles.cookiePolicy}>
+                            <div>
+                                After a successful sign in, we use a cookie in your browser to track your session.&nbsp;
```

Comment:
If we have added internalization (i18n) then these messages need to internalized.

Reply:
i18n is not implemented in these apps.